frontend/src/features/env/EnvService.ts

```diff
@@ -88,17 +88,10 @@ export const submitSwitchEnvFormFeedback = async ({
 }): Promise<SuccessMessageDto | ErrorDto> => {
   const isAdmin = Object.keys(formInputs).includes('rating')
 
-  const formFields = [
-    'url',
-    'feedback',
-    'email',
-    'rumSessionId',
-    'attachmentType',
-    'userAgent',
-    'responseMode',
-    'authType',
-  ]
+  const formFields = ['url', 'feedback', 'email', 'rumSessionId']
   if (isAdmin) formFields.push('rating')
+  if (!isAdmin)
+    formFields.push('attachmentType', 'userAgent', 'responseMode', 'authType')
```

Comment:
Another nit, should we abstract this to make it more verbose?

like 
```
adminFormFields = ['rating']
publicFormFields = ['attachmentType', 'userAgent', 'responseMode', 'authType']
if (isAdmin) formFields.push(...adminFormFields)
else formFields.push(...publicFormFields)
```

Reply:
[20bd598](https://github.com/opengovsg/FormSG/pull/5981/commits/20bd598415531c3246827887baed86276c104147)